models/topics.models.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../db/connection")
+
+
+const fetchTopics = (queries) => {
+    const description = queries.description;
+    const slug = queries.slug;
+
+    let SQLString = `SELECT * FROM topics`
+    const args = []
+
+    return db.query(SQLString, args)
+    .then(({ rows}) => {
+        console.log(rows, "<<<< rows in fetchTopics, line 12");
+        return rows 
+    })
+    .catch((err) => {
+        console.error("Error fetching topics:", err);
+        throw err;
+    });
+}
+
```

Comment:
You are only fetching the topics here, there's no need to complicate things with additional query parameters or error handling.